feature.env

```diff
@@ -21,3 +21,4 @@ SYSTEM_CLIENT_ID=clientid
 SYSTEM_CLIENT_SECRET=clientsecret
 
 UPDATE_INDUCTION_TRAINING_INTERESTS_IN_PRISON_SECTION_ENABLED=true
+UPDATE_INDUCTION_SKILLS_AND_INTERESTS_SECTION_ENABLED=true
```

Comment:
I know this isn't part of this PR, but I'm not sure about the one above. Here's where it's used:

```
trainingAndInterestsInPrisonSectionEnabled: toBoolean(
   get('UPDATE_INDUCTION_TRAINING_INTERESTS_IN_PRISON_SECTION_ENABLED', false, requiredInProduction),
),
```

And also:

```
{%- if featureToggles.induction.update.trainingAndInterestsInPrisonSectionEnabled -%}
   /prisoners/{{ prisonerSummary.prisonNumber }}/induction/in-prison-work
{%- else -%}
```

Shouldn't it be `UPDATE_INDUCTION_IN_PRISON_WORK_INTERESTS_SECTION_ENABLED` and `featureToggles.induction.update.inPrisonWorkInterestsSectionEnabled` or `UPDATE_INDUCTION_IN_PRISON_INTERESTS_SECTION_ENABLED` and `featureToggles.induction.update.inPrisonInterestsSectionEnabled` (depending on whether we need to distinguish between `work` and `training`, or we just have `in-prison interests` in general).

Reply:
Yeah, I agree there is cross over and overloaded terms

Given that we agreed to migrate the Update functionality **section** at a time, where a "section" is how the PLP UI renders/groups questions, these feature toggles are named with the word "section" in them to try to make this clear (perhaps we're not hitting the mark there! 🤣 )
But its to do with the logical sections on the PLP UI, rather than the individual questions, or groupings within the API

Does that answer it? - its basically for this section in the PLP UI
![Screenshot 2024-02-21 at 10 35 14](https://github.com/ministryofjustice/hmpps-education-and-work-plan-ui/assets/94835226/1fdc795d-d845-4c9a-a4e0-d9fe153301b0)

Reply:
Ok, I've looked at the screens and we have:

![image](https://github.com/ministryofjustice/hmpps-education-and-work-plan-ui/assets/135589132/e386f3d5-2a0d-4cf0-a0bb-a6f783781cae)

Plus:

![image](https://github.com/ministryofjustice/hmpps-education-and-work-plan-ui/assets/135589132/8d7b01fb-28f0-4556-89d5-aa6594d0ce07)

So we have 2 feature toggles to cover this: `UPDATE_INDUCTION_TRAINING_INTERESTS_IN_PRISON_SECTION_ENABLED` (for the relevant table on the `education-and-training` tab) and `UPDATE_INDUCTION_WORK_INTERESTS_SECTION_ENABLED` (for the relevant table on the `work-and-interests` tab). Ok that makes more sense. It's just that the latter will cover both "in-prison" and "post-prison" work interests.

This terminology is confusing, but I think it's probably ok now that I understand the mismatch!